solr/solrj-streaming/src/java/org/apache/solr/client/solrj/io/stream/LetStream.java

```diff
@@ -224,4 +224,9 @@ public StreamComparator getStreamSort() {
   public int getCost() {
     return 0;
   }
+
+  @SuppressWarnings({"rawtypes"})
+  public Map getLetParams() {
+    return this.letParams;
+  }
```

Comment:
This method has no usage, is it needed? And if so, should we not simply add the types to `Map<String, Object>` rather than suppressing them?